lib/mock-render/mock-render.ts

```diff
@@ -5,7 +5,8 @@ import { directiveResolver } from '../common/reflect';
 
 function MockRender<MComponent, TComponent extends {[key: string]: any}>(
   template: string | Type<MComponent>,
-  params?: TComponent
+  params?: TComponent,
+  detectChanges = true
```

Comment:
maybe as an object in case if we need to add more flags later?

Reply:
I had that though too...but couldn't think of other options. My thought was that it was easy enough to write an overload/union if we were to pivot to an object.

Reply:
agree

Reply:
Welcome back BTW :-D